surfsense_backend/app/agents/researcher/graph.py

```
@@ -11,12 +11,27 @@ class GraphState(TypedDict):
     # Final output
     final_written_report: Optional[str]
 
+def route_based_on_research_mode(state: State) -> str:
```

Pylint (3.3.7)

[warning] Unused argument 'state'

(W0613)